compat/cpu.h

```diff
@@ -14,70 +14,9 @@
 #endif
 #if HAVE_SCHED_H
 #include <sched.h>
+#else
+#include "compat/sched_affinity.h"
```

Comment:
There is no need to keep an empty compat/cpu.h file around. The files using it should include compat/sched.h instead.

FYI, if you did this as a rename-then-edit it may help clarify which code you are actually changing than telling us the entire file contents here are deleted and whole new files added elsewhere (though some git tooling is not that smart AFAIK github is).